splash/qtrender_lua.py

```diff
@@ -449,10 +449,37 @@ def set_custom_headers(self, headers):
         self.tab.set_custom_headers(self.lua2python(headers, max_depth=3))
 
     @command()
+    def get_viewport_size(self):
+        sz = self.tab.web_page.viewportSize()
+        return {'width': sz.width(), 'height': sz.height()}
+
+    @command(table_argument=True)
```

Comment:
See http://splash.readthedocs.org/en/latest/scripting-tutorial.html#calling-splash-methods and https://github.com/scoder/lupa#python-callables